.github/workflows/test.yml

```diff
@@ -147,10 +147,24 @@ jobs:
         # Example of how to exclude a fully qualified test:
         # - {"ipfamily": {"ip": ipv4}, "ha": {"enabled": "false"}, "gateway-mode": shared, "target": {"shard": shard-n-other}}
         exclude:
+         # Not currently supported but needs to be.
          - {"ipfamily": {"ip": dualstack}}
-         # Not currently supported
          - {"ipfamily": {"ip": ipv6}, "target": {"shard": control-plane}}
          - {"ipfamily": {"ip": ipv6}, "gateway-mode": shared}
+         # Limit matrix combinations for CI. DISABLED items added to exclude list:
```

Comment:
per commit msg "Limit CI test matrix per issue #1401"

Use Closes, Fixes etc in the commit msg body instead (see other comment with the link)